From akpm@linux-foundation.org  Wed Aug 15 16:34:22 2007
From: Mariusz Kozlowski <m.kozlowski@tuxland.pl>
Date: Fri, 10 Aug 2007 14:53:37 -0700
Subject: usb: remove redundant memset from amd5536udc
To: greg@kroah.com
Cc: linux-usb-devel@lists.sourceforge.net, akpm@linux-foundation.org, m.kozlowski@tuxland.pl
Message-ID: <200708102153.l7ALrc0K011554@imap1.linux-foundation.org>


From: Mariusz Kozlowski <m.kozlowski@tuxland.pl>

Remove redundant memset() call from udc_pci_probe().  No functional change.

Signed-off-by: Mariusz Kozlowski <m.kozlowski@tuxland.pl>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/amd5536udc.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/usb/gadget/amd5536udc.c
+++ b/drivers/usb/gadget/amd5536udc.c
@@ -3244,7 +3244,6 @@ static int udc_pci_probe(
 		retval = -ENOMEM;
 		goto finished;
 	}
-	memset(dev, 0, sizeof(struct udc));
 
 	/* pci setup */
 	if (pci_enable_device(pdev) < 0) {
